tests/api/test_category.py

```diff
@@ -108,8 +144,60 @@ def test_category_create_mutation(
     assert data['category']['parent']['id'] == parent_id
 
 
+def test_category_create_mutation_without_background_image(
+        monkeypatch, staff_api_client, permission_manage_products):
+    query = """
+        mutation($name: String, $slug: String, $description: String, $parentId: ID) {
+            categoryCreate(
+                input: {
+                    name: $name
+                    slug: $slug
+                    description: $description
+                },
+                parent: $parentId
+            ) {
+                category {
```

Comment:
Do we need to fetch all of the fields for the category? It will be the same as in other tests, could we test just the minimal scope necessary?